samples/snippets/csharp/VS_Snippets_Remoting/System.Net.Authentication/CS/custombasicauthentication.cs

```diff
@@ -124,7 +124,7 @@ private static void getPage(String url)
 
     // The displayPageContent method display the content of the
     // selected page.

```

Comment:
Make sure to update the naming in the comments too. For this method and the other ones.
Also, I think there is a parallel VB and C++ sample.
There is also another sample in System.Net.Sockets.Authentication